src/drtsans/stitch.py

```diff
@@ -242,6 +280,7 @@ def stitch_binned_profiles(iq1d_unbinned, iq1d_binned, reduction_config):
         return iq1d_combined_out
 
     boundaries = get_stitch_boundaries(reduction_config, iq1d_unbinned, anisotropic=len(iq1d_binned[0]) > 1)
+    target_profile_index = get_target_profile_index(iq1d_binned, reduction_config)
```

Comment:
variable `target_profile_index` is used only once in line 294 so I'd just remove it, and use
`get_target_profile_index(iq1d_binned, reduction_config)` at line 294